internal/services/mobilenetwork/helpers.go

```diff
@@ -42,3 +43,41 @@ func resourceMobileNetworkChildWaitForDeletion(ctx context.Context, id string, g
 
 	return nil
 }
+
+// some resources defined both systemAssigned and userAssigned Identity type in Swagger but only support userAssigned Identity,
+// so add a workaround to convert type here.
```

Comment:
@ziyeqf is there a Swagger bug for this?

Reply:
just opened one https://github.com/Azure/azure-rest-api-specs/issues/22634